rollup.config.js

```diff
@@ -16,7 +16,22 @@ const copyStatic = copy({
 	targets: [
 		{ src: 'types', dest: 'dist' }
 	]
-})
+});
+
+const babelConfig = {
+    exclude: [/\/core-js\//],
+    presets: [
+        [
+            "@babel/preset-env",
+            {
+                targets: '> 0.1% or not dead',
+                useBuiltIns: 'usage',
+                corejs: 3
+            }
+        ]
+    ],
+    plugins: ["css-modules-transform"]
+};
```

Comment:
This is actually a good idea, I'll incorporate this into https://github.com/visjs/vis-network/pull/85 (mainly a note to myself).

However you have to add `babelrc: false`to achieve what you claimed. Otherwise local .babelrc will be used in dependencies and defeat the purpose of this.

Source: https://github.com/rollup/rollup-plugin-babel#external-dependencies

Also I would rather see this in .babelrc and being loaded here as there are more things using Babel than Rollup. For example tests now fail with:
```
import util from 'vis-util';
       ^^^^

SyntaxError: Unexpected identifier
```
which is perfectly valid syntax.

PS: Please always run `npm t` to make sure you didn't break anything before opening a PR, thanks.

Reply:
> PS: Please always run npm t to make sure you didn't break anything before opening a PR, thanks.

Hi @Thomaash, I'm now executing `npm test` on the master branch as is and I got errors:

```console
npm ERR! code ELIFECYCLE
npm ERR! errno 49
npm ERR! vis-timeline@0.0.0-no-version test: `mocha --exit --require @babel/register`
npm ERR! Exit status 49
npm ERR!
npm ERR! Failed at the vis-timeline@0.0.0-no-version test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<my user>\AppData\Roaming\npm-cache\_logs\2019-09-18T13_56_53_888Z-debug.log
```
Am I missing something to setup on my environment to run tests? 

PS: I have not much experience with npm.

Reply:
Does the following pass?
```shell
git fetch --all && git checkout upstream/master && npm ci && npm t
```

PS: 49 tests should fail and 1 pass.
PPS: I assume you have `git@github.com:visjs/vis-timeline.git` as upstream.

Reply:
> PS: 49 tests should fail and 1 pass.

Yeah, that's what I got. I though that (having so many fails) was an error, but then is OK now. I push a commit to restore .babelrc and for adding babelrc: false to babelConfig and then ran tests and I got the same as upstream master.